includes/batch.inc

```diff
@@ -143,7 +143,7 @@ function _batch_progress_page_nojs() {
     print $fallback;
 
     // Perform actual processing.
-    list($percentage, $message) = _batch_process($batch);
+    list($percentage, $message) = _batch_process();
```

Comment:
Just a note for whoever happens to come across it: this is indeed a proper change. $batch was never used as an argument to _batch_process(), at least not in D6's lifetime. _batch_process() gets the batch not from the function argument but from its first line
```
$batch =& batch_get();
```